Library/ViewModels/ManagePledgeViewModel.swift

```diff
@@ -99,6 +101,10 @@ public final class ManagePledgeViewModel:
 
     self.rewardReceivedViewControllerViewIsHidden = projectAndReward
       .map { project, reward in reward.isNoReward || project.personalization.backing?.status != .collected }
+
+    self.showSuccessBannerWithMessage = self.pledgeViewControllerDidUpdatePledgeSignal.mapConst(
```

Comment:
Is the idea behind putting this string in `ManagePledgeViewModel` instead of `PledgeViewModel` (where the update pledge logic happens) that we're anticipating reusing this string for other "update" types? (ie. update payment method, update reward).

Reply:
No, just that this is a side-effect from the delegate being notified about the success of the request so that the banner is visible when the VC is popped off the stack.